### DISCUSSION TITLE
What formats of virtual slides are supported?

### DISCUSSION THREAD

**Discussion by nedash** [camicroscope/caMicroscope#977] [General]
We tested Aperio (.svs) and all works fine. And what about other formats? Especially, when slide consist of several files. I could not found found any related information in the documentation. Thanks.

**Comment by birm** [comment]
We currently believe that we have a list of formats which seem to be at least likely to work: https://github.com/camicroscope/caMicroscope/blob/a9058bd34392c9398625b669d795f7d0500e7ff9/apps/table.js#L16
These should be the single-file formats from bioformats and openslide (plus dicom series).
Usually, these have to have a pyramid built in, but this may vary.

I haven't tested very extensively, so please let us know if you have issues with any files you want to try.

**Comment by nedash** [comment]
Thanks for the feedback.
